content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+to expire. Enter the number of days before a contract is going to expire that an alert email will be
+sent.
+
+.. note::
+   To determine who the responsible person is for a contract, open an individual contract. The
+   person listed as :guilabel:`Responsible` under the :guilabel:`Contract Information` section of
+   the contract is the person who will receive the alert.
+
+   To access all contracts, navigate to :menuselection:`Fleet app --> Fleet --> Contracts` and all
+   contracts appear in the list. Click on a :guilabel:`Contract` to view it.
+
+   An individual contract can also be found by navigating to :menuselection:`Fleet app --> Fleet -->
+   Fleet` and clicking on an individual vehicle. On the vehicle form, click the
+   :guilabel:`Contracts` smart button at the top of the page. The contract(s) associated with this
+   vehicle only appears in the list. Click on an individual contract to open it. The
+   :guilabel:`Responsible` person is listed on the contract.
+
+New Vehicle Request
+-------------------
+
+Set a limit on how many vehicle are in the fleet by setting a limit on requests. When using the
+salary configurator, employees will not be able to request a new car if there is a specific amount
+of available cars already in the fleet that do not have a current driver assigned to them. Enter the
+number limit for existing available cars in this field.
+
+.. image:: configuration/fleet-settings.png
+   :align: center
+   :alt: Settings available for the Fleet application.
+
+Manufacturers
+=============
+
+Odoo Fleet comes pre-configured with sixty-six commonly used car manufacturers in the database,
+along with their logos. To view the pre-loaded manufacturers, go to :menuselection:`Fleet app -->
+Configuration --> Manufacturers`.
+
+The list of manufacturers appear in alphabetical order. Each manufacturer's card lists how many
+specific models are configured for each particular manufacturer. Odoo comes with forty five
+pre-configured models for four major auto manufacturers.
+
+.. image:: configuration/manufacturer.png
+   :align: center
+   :alt: Manufacturer card with the amount of models listed.
+
+Add a manufacturer
+------------------
+
+To add a new manufacturer to the database, click :guilabel:`Create`. A manufacturer form will
+load. Only two pieces of information are needed, the name of the manufacturer, and the logo. Type
+the name of the manufacturer in the name field, and select an image to upload for the logo. When the
+information is entered, click :guilabel:`Save`.
+
+Vehicle Models
+==============
+
+Each manufacturer has a variety of models that they produce. When adding a vehicle to the fleet, it
+is important to specify the vehicle being added. Odoo comes with pre-configured models for four
+manufacturers, so it is likely new models will need to be added to the database to correctly
+configure a fleet of vehicles.
```

Comment:
This sentence doesn't make sense-- if they come preconfigured why do I have to add them?

Reply:
I updated this so it's more clear- because there's only models configured for 4 manufacturers-most people (outside of Belgium) will need to add models.